test/test_foreach.py

```diff
@@ -646,6 +646,20 @@ def test_foreach_l2_large_value_input(self, device, dtype, op):
                 for i, e in enumerate(expect)))
         self.assertEqual(expect, actual, equal_nan=False)
 
+    @onlyCUDA
+    @ops(foreach_reduce_op_db, allowed_dtypes=floating_types())
+    def test_foreach_norm_big_num_tensors(self, device, dtype, op):
```

Comment:
Why is this only running foreach_norm give that it is decorated with "foreach_reduce_op_db"?

Reply:
foreach_norm is our only reduce op but yes i see your point + will remove!